<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>545</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Отправить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QPushButton" name="pushButton_6">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Бригада 4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Бригада 5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QPushButton" name="pushButton_8">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Бригада 6</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>235</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Бригада 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_5">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Бригада 3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>200</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Бригада 2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>240</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Алексеев Никита Евгеньевич</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Анюков Антон Васильевич</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Аюпов Олег Максимович</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Белякова Анна Артемовна</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Боброва Валентина Богдановна</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Васильев Илья Григорьевич</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Горбачёва Инга Федоровна</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Дубина Оксана Павловна</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Евдокимова Ольга Петровна</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Зайцев Мирослав Константинович</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Зимин Егор Матвеевич</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Знойный Глеб Андреевич</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Карпов Никита Сергеевич</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Кошелева Майя Дмитриевна</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Максимова Софья Аркадьевна</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Мурина Светлана Алексеевна</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Николаев Корней Кириллович</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Петрова Анастасия Станиславовна</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Рудников Павел Федотович</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Степанов Пантелеймон Маркович</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Степанова Северина Всеволодовна</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Суханова Симона Степановна</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Сырчикова Ева Ильинична</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Уварова Елена Михайловна</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>был на смене</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>была на смене</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>не был на смене</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>не была на смене</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_9">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Отослать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Подключиться к серверу</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
